src/examples/java/io/nats/examples/NatsJsPushSubQueue.java

```diff
@@ -0,0 +1,159 @@
+    static final String IDENT = "" + System.currentTimeMillis(); // so you can re-run without restarting the server
+    static final String STREAM = "jsq-stream-" + IDENT;
+    static final String SUBJECT = "jsq-subject" + IDENT;
+    static final String QUEUE = "jsq-queue-" + IDENT;
+    static final String DURABLE = "jsq-durable-" + IDENT;
+
+    static final int MSG_COUNT = 100;
+    static final int NO_OF_SUBSCRIBERS = 5;
+    static final Duration TIMEOUT = Duration.ofMillis(500);
+
+    public static void main(String[] args) {
+        try (Connection nc = Nats.connect(ExampleUtils.createExampleOptions(args, true))) {
+
+            // Create the stream.
+            NatsJsUtils.createOrUpdateStream(nc, STREAM, SUBJECT);
+
+            // Create our JetStream context
+            JetStream js = nc.jetStream();
+
+            // Setup the subscribers
+            // - the PushSubscribeOptions can be re-used since all the subscribers are the same
+            // - use a concurrent integer to track all the messages received
+            // - have a list of subscribers and threads so I can track them
+            PushSubscribeOptions pso = PushSubscribeOptions.builder().durable(DURABLE).build();
+            AtomicInteger allReceived = new AtomicInteger();
+            List<JsQueueSubscriber> subscribers = new ArrayList<>();
+            List<Thread> subThreads = new ArrayList<>();
+            for (int id = 1; id <= NO_OF_SUBSCRIBERS; id++) {
+                // setup the subscription
+                JetStreamSubscription sub = js.subscribe(SUBJECT, QUEUE, pso);
+                // create and track the runnable
+                JsQueueSubscriber qs = new JsQueueSubscriber(id, js, sub, allReceived);
+                subscribers.add(qs);
+                // create, track and start the thread
+                Thread t = new Thread(qs);
+                subThreads.add(t);
+                t.start();
+            }
+            nc.flush(Duration.ofSeconds(1)); // flush outgoing communication with/to the server
+
+            // create and start the publishing
+            Thread pubThread = new Thread(new JsPublisher(js));
+            pubThread.start();
+
+            // wait for all threads to finish
+            pubThread.join();
+            for (Thread t : subThreads) {
+                t.join();
+            }
+
+            // report
+            for (JsQueueSubscriber qs : subscribers) {
+                qs.report();
+            }
+        }
+        catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    static class JsPublisher implements Runnable {
+        JetStream js;
+
+        public JsPublisher(JetStream js) {
+            this.js = js;
+        }
+
+        @Override
+        public void run() {
+            for (int x = 1; x <= MSG_COUNT; x++) {
+                try {
+                    js.publish(SUBJECT, ("Data # " + x).getBytes(StandardCharsets.US_ASCII));
```

Comment:
Perhaps add a comment that the publish ack is ignored.  Examples are copied, and want to remind the copier.  :)